.github/workflows/stale-branches.yml

```diff
@@ -0,0 +1,18 @@
+name: Stale Branches
```

Comment:
Channeling @Josmithr... please add a header comment with the intent of the action, where to get more info or something, etc. pasting your PR description might be appropriate.